index.html

```diff
@@ -9,6 +9,45 @@
     <div id="app">
       <p>Hello World! Open the console to see if Klicky is running</p>
     </div>
+    <div class="pika">
+      <img src = './src/pika.gif' width = "500" id="movin">
```

Comment:
Nice job figuring out how to add an image! 2 notes:

- There is an 'assets' folder that you can use to drop images, better to keep all the images in 1 folder than to create another one
- Seems like `pika.gif` doesn't actually exist at `./src/pika.gif`, but instead `./src/pictures/pika.gif`, when I checkout your code, it won't build because of that :(